src/h3.ts

```diff
@@ -74,6 +74,7 @@ export const H3Core = /* @__PURE__ */ (() => {
           handlerRes = this.handler(event);
         }
       } catch (error: any) {
+        console.error(error);
```

Comment:
I think we could move it to [here](https://github.com/h3js/h3/blob/main/src/response.ts#L59) and log any unhandled error